Automattic/pocket-casts-android
# Anonymous Bump Stats (part 1)

## Description

This adds the ability to anonymously bump a stat. We need this for purposes of allowing users to opt-out of analytics while still allowing us to get the data we need to be able to provide to paid sponsors. 

The only data that is provided with these events are:
1. An event name
2. The time of the event
3. Any specific properties about the event (i.e., the id of the list that the user interacted with in the discover feed)

The second part of this work will involve providing anonymous daily unique stats, which I will address in a follow-up PR. This sets us up to drop Firebase as an analytics provider, and allows us to do a better job of restricting the data we collect.

## Testing Instructions
1. Load the app
2. Go to the discover screen and perform the the following actions:
    1. Subscribe to a podcast
    2. Tap on "Show All" to see all of a list
    6. Play a featured episode from the Discover Screen
    7. Tap on a podcast on the Discover screen
    8. Tap. on an episode on the Discover screen
3. Put the app in the background and reload it
4. Observe that the relevant events are sent to Tracks

## Checklist
 
#### I have tested any UI changes...
